src/CalcViewModel/Common/Utils.h

```diff
@@ -11,6 +11,8 @@
 // Utility macros to make Models easier to write
```

Comment:
Can we remove OBSERVABLE_PROPERTY_RW_ALWAYS_NOTIFY from this file?

Reply:
done